mne_bids/write.py

```diff
@@ -1032,38 +1032,36 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
                       .format(t1w_basename))
 
     # Check if we have necessary conditions for writing a sidecar JSON
-    if trans is None:
-        return anat_dir
+    if trans is not None:
```

Comment:
I will merge as it just seems cosmetic but why did you nest this? Zen of Python says:

"Flat is better than nested"

Reply:
It felt more readable :thinking: ... and it allowed me to get rid of one `return` statement.

But I agree, perhaps having it flat would have been better :-|